### ISSUE TITLE
Looping Events don't stop

### ISSUE THREAD

**Issue by shawnwignall** [Avindr/MxM-IssueTracking#83]
I'm unable to stop a looping event.  I have tried the EndLoopEvent, but the looping animation just continues on forever.

Using MxM version 2.2.7 with the hotfix, the latest Strider, and Unity 2020.1.2

**Comment by shawnwignall** [comment]
New Info:  If I trigger another event, I can escape the looping event.  If I just try to go back to normal movement, the looping animation doesn't end.

**Comment by Avindr** [comment]
Hi, I've just tested this and it seems to be working as intended. Though there may be some confusion in the documentation where it wasn't quite clear how to do this. I've now rectified that in the coding with events section of the documentation. 

There are two ways to stop looped events and they both have a slightly different result:

MxMAnimator.ForceExitEvent();    <- This one will cancel the event immediately and move straight back into motion matching
MxMAnimator.EndLoopEvent();    <- This one will stop the event from looping but it will still play until the end of this loop.
